### PR TITLE
Gaming Tournament Site

### PR DESCRIPTION
**Pull request by Chimwemwe-127001**
## Gaming Tournament Site
In this website update, the following requirements have been implemented:
- [x] Personalize page content.
- [x] Follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
- [x] Pages should look almost identical to the original design. Room for small adjuments, allowed.
- [x] Build Home page and About page.
- [x] Page versions (Mobile: up to 768px wide.) (Desktop: 768px or wider.)
- [x]  Interactions:
- [x] Dynamic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,12 @@
 > This capstone project was built for Microverse [under these guidance objectives](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) > _note: you can only access this link if you're a Micronault (Microverse Student)_
 
 ## Live Demo
->   You can access the live demo of this repo [here]()
+>   You can access the live demo of this repo [here](https://chimwemwe-127001.github.io/module_1_capstone/)
 
-![gif screenshot](./assets/img/Desktop_Version_Preview.gif)
+![screenshot](./assets/images/Desktop_preview.png)
```
Comment (carlos-ssh) [inline README.md:14]: - Provide a video link with maximum length of the video is 5 minutes in professional English
- the video should contains a demo of the project, showing the pages in different screen sizes and 
- the video contains a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Chimwemwe-127001 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

changesrequested

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Chimwemwe-127001) [reply README.md:14]: Thank you very much @carlos-ssh for the review. I have made the required changes as requested. Hopefully, my second attempt will be approved.
Happy Coding!!😁👨‍💻

Comment (henatan99) [reply README.md:14]: This issue is well addressed. Thank you for making the requested changes. :clap: :+1:

File: style.css [hunk 2]
```diff
@@ -0,0 +1,783 @@
...
+
+.hamburger.open::after {
+  transform: translateY(-3px) rotate(-90deg);
+}
+
+.welcome-section {
+  background-image: url(./assets/images/26293.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```
Comment (henatan99) [inline style.css:71]: - [ ] OPTIONAL: Personalizing is meant for the contents only not the design requirements. You should always consider design requirements as the client's business requirements and you should make your design as close as the original design. It includes the background color, typography (color, font style, font-family, font-size), spacing, positioning, color and size contrast, etc... Kindly look at the attached screenshot below just as a reference and take a lesson for your future work. :pray: 

image
image

**Review by henatan99 — APPROVED** [review]
Hi @Chimwemwe-127001, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Chimwemwe-127001) [reply README.md:14]: > Thank you very much @henatan99, for addressing the issue...👍

File: style.css (hunk 2 above)
Comment (Chimwemwe-127001) [reply style.css:71]: - Thank you a lot for this insightful review 🤓 . I am currently working to ensure this is rectified. This has really helped me take a deep understanding of clients' business requirements. I will surely take this lesson for my future work.
- Thank you once more for the review. 
HAPPY CODING!!!😁👨‍💻👨‍💻

**Pull request merged**